### ISSUE TITLE
error with eventmanager

### ISSUE THREAD

**Issue by BogusCurry** [watcher64/osCore2#2]
Hi there ;D
I run the OSCore2 local on my pc unter Linux (LinuxMint 18-3) and the mono 5.10 version ;D Now I have error message in the console from eventmanager
https://paste.md-5.net/gebetuvile.avrasm

And when I log in, so is my avatar a cloud and I have very big lag, the inventory is not loading. maybe I have found a bug or must I more edit the opensim.ini ? 

Greetings
Bogus

**Comment by watcher64** [comment]
Yes as the readme says, if you are not using jopensim profiles and search remove those DLL's

**Comment by watcher64** [comment]
Oops , the readme updates did not make it to Demetrius , they are only on the master

**Comment by watcher64** [comment]
I really need to add more to this, you should compile it with msbuild not nant, not sure how you compiled it, nant does not fully support .net 4.6 builds and this codebase was pushed to 4.6 ..

I run my grid on it, and don't have any issues, but I'm always happy to help when they are found ..

**Comment by BogusCurry** [comment]
under Linux with msbuild ? I make this for compiled : xbuild OpenSim.sln ok, before this I make rebuild.sh ;D The actual mono version is compatibel to 4.6 ? or little bit 4.6 ? lol I not know it. So I see you are in the same Timezone as I ? Know you Discord or IRC ? Oh when you run a grid, how is the name of the grid ?

**Comment by watcher64** [comment]
yes under linux with msbuild, it should be included int he mono package ..

I'm actually in Eastern time, US, but yes I have a discord channel ..

https://discord.gg/xXGWhxJ

**Comment by watcher64** [comment]
I also hang out in #opensim-dev and #osCore2 on IRC .. BillBlight

**Comment by BogusCurry** [comment]
ahh ok ;D I am now in your grid opensim.life ;D 
Ok, here is very late, so I must go sleep ;D Maybe I comes tomorrow in your discord channel ;D 

good night ;D
